content/docs/studio/rest-api.md

```diff
@@ -0,0 +1,60 @@
+# REST API
+
+The purpose of Studio REST API is to give programmatic access to information in
+Studio and executing actions in it.
+
+The API is hosted under the `/api` route on the Studio server:
+https://studio.iterative.ai/api or https://your-domain/api in case of
+[self-hosted Studio](/doc/studio/self-hosting/installation).
+
+To use API, you need to generate
+[Studio access token](/doc/studio/user-guide/account-management#studio-access-token).
+
+## Download model
+
+Get signed url to download the model binaries for a model from Model Registry.
+Requires the model to be stored with DVC with s3 or azure remote. Note, that you
+need to
+[set up remote cloud credentials](/doc/studio/user-guide/account-management#cloud-credentials)
+for Studio have rights to sign urls.
+
+```yaml
+Endpoint: api/model-registry/get-download-uris
+HTTP Method: GET
+```
+
+### Request
+
+| param   | desc          | type   | required | example value                      |
+| ------- | ------------- | ------ | -------- | ---------------------------------- |
+| repo    | Git repo URL  | string | true     | iterative/demo-bank-customer-churn |
```

Comment:
```suggestion
| repo    | Git repo URL  | string | true     | git@github.com:iterative/demo-bank-customer-churn |
```

Does this also support a Studio project ID per https://github.com/iterative/studio/pull/6338#issuecomment-1596468450? My feeling is it still might be useful since users have to do a lookup either way and this requires less escaping than a full repo url.

Reply:
Not rn. But we can add it in some future. Adding this as an item to "out of scope of phase I" https://github.com/iterative/studio/issues/5177